docs/SEEEDSTUDIO_WIO_E5.md

```diff
@@ -83,18 +83,20 @@ The first time that you flash the mLRS firmware to either the E5 Mini or E5 Grov
 - Connect your ST-Link to the 3V3, GND, SWD, and SWCLK pins on the board
     - The SWD and SWCLK pins are labeled 'DIO' and 'CLK' respectively
     - Some ST-Link boards provide power and some do not - ensure the E5 is powered
-- Launch STM32CubeProgrammer and select ST-Link as the connection method, click connect
+    - While applying power also hold the boot button or ground out the boot pad for the Wio E5-Grove, this will ensure initial connection with STM32CubeProgrammer
```

Comment:
I'm not totally sold on this. I have never found me doing this.
IMHO that's the kind of suggestion which maybe some may find but others may not find.
That is, I guess, it could be a point to try under a hypothetical section "troubleshooting", but not as a general step under "this is what you must do"

Reply:
I was unable to get any connetion to stm32 to change the readout protection unless I held the boot button while applying power. Both the mini and grove did this. once readout was changed this is no longer needed.

Reply:
I perfectly understand that it was the procedure which brought you to success; this is not desputed ... the point I'd like to make is that IMHO it is not the general approach which every user must do to reach success.
At this point in time I think we don't have a statistics on how many have to do that and how many don't, so we don't know at this point in time if this is the rare exception or if it is what a majority need to do.
I think that a doc should not be filled with mandatory steps which happen to be needed only for rare cases, especially if these steps require some level of effort (I would not like to solder that wires/button if I actually don't have to :))

This are however just my 2 cents. Ultimately you guys, especially @jlpoltrack, should decide :)